okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -243,6 +245,30 @@ public final OkHttpChannelBuilder sslSocketFactory(SSLSocketFactory factory) {
   }
 
   /**
+   * Set the hostname verifier to use when using TLS negotiation. The hostnameVerifier is only used
+   * if an sslSocketFactory is set. If the hostname verifier is not set, a default hostname
```

Comment:
> The hostnameVerifier is only used if an sslSocketFactory is set.

Actually, that's a bit misleading. You can use TLS without manually setting the `sslSocketFactory()`; grpc will just create one for you (in `createSocketFactory()`).